src/components/svg/HomeSvg.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+export default function HomeSvg() {
```

Comment:
Nice move ! :) 

Optionally you could use webpack loader instead importing svg's the same way later as you do with this approach. We can talk about it separately.